src/main/webapp/app/add-event.jsp

```diff
@@ -2,47 +2,86 @@
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
 <html>
 <head>
-    <title>Add Event</title>
+    <title>Add/Edit Event</title>
+    <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+    <link rel="stylesheet" href="/resources/demos/style.css">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
     <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
     <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
 </head>
-<body>
+<body onload="checkFunction()">
 <style>
     textarea {
 
-        height: 20px;
-        width: 200px;
+        height: 50px;
+        width: 400px;
     }
+    #buttons{
+        position: absolute;
+        bottom: 0;
+        width:100%;
+        height:60px;
+
+    }
+    #add,#edit{
+        border: dotted;
+        text:bold;
+        text-align: center;
+    }
+
 </style>
-<header>Add New Event</header>
-<form method="post">
+<header id="add" class="w3-hide"><h1>Add New Event</h1></header>
+<header id="edit" class="w3-hide">Edit Event</header>
+<form name="eventform" method="post" style="padding: 15px">
     <p><b>Name of Event</b></p>
-    <p><textarea name="text" id="name"></textarea></p>
+    <p><textarea name="name" id="name"></textarea></p>
     <p><b>Description</b></p>
-    <p><textarea name="text" id="description"></textarea></p>
-    <p><b>Date</b>></p>
-    <p><input type="text" id="datepicker"></p>
+    <p><textarea name="desc" id="description"></textarea></p>
+    <p><b>Agenda</b></p>
+    <p><textarea name="agenda" id="agenda"></textarea></p>
+    <p><b>Date</b></p>
+    <p><input name="date" id="datepicker"></p>
 </form>
-<p>
+<div id="buttons" >
     <button type="submit" onclick="saveDataToDB()">Save</button>
     <button onclick="goToTheMainPage()">Cancel</button>
 
-</p>
+</div>
 </body>
 
 <script>
+    var data = {};
```

Comment:
Please keep function code in a separate JS file, so as to not overload JSP files with logic.